lib/renuo/bin-check/command_result.rb

```diff
@@ -0,0 +1,10 @@
+module RenuoBinCheck
+  class CommandResult
+    attr_accessor :output, :error_output, :exit_code
```

Comment:
would a attr_reader be enough? then the result would become a value object (immutable), which is often a good idea https://en.wikipedia.org/wiki/Value_object http://c2.com/cgi/wiki?ValueObject